content/blog/dotnet-preview/index.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Infrastructure as Code with .NET and Pulumi"
+authors: ["sophia-parafina"]
+tags: [". NET", "C#", "F#", "Visual Basic"]
+meta_image: "microsoft_logo.jpg"
+meta_desc: "Pulumi supports . NET languages for Infastructure as Code"
+date: "2019-11-08"
+---
+
+We are excited to announce the preview of Pulumi . NET support. Pulumi lets you declare infrastructure using general purpose programming languages. Our goal is to automate applications at all layers and help developers and operators work better together by eliminating the silos that separate them.
+
+Millions of . NET developers can now use their favorite languages and open source ecosystems to build modern, cloud native applications. We've added support for C#, F# and Visual Basic. Because the . NET framework is available on Windows, Linux and macOS you have a choice of platforms to use. You can create cloud resources by writing **Microsoft . NET Core** programs to build and deploy AKS Clusters, Functions, Azure App Services, Virtual Machines, Cosmos DBs, KeyVaults, and much, much more.
+
+## What Does Pulumi Enable?
+
+As a . NET developer you can:
+
+* Declare infrastructure in C#, F#, or Visual Basic NET.
+* Code with favorite IDEs, including Visual Studio, Visual Studio Code, and Rider, with auto-completion and refactoring tools.
+* Catch mistakes early on with red markers and build errors.
+* Allow package managers to reference any NuGet library compatible with . NET Core 3.0.
+* Use the Pulumi CLI or CI/CD integrations to perform deployments predictably and reliably.
+* Maintain your infrastructure in a repository and version it
+* Build modern, reliable, and scalable applications using a multitude of cloud architectures, including containers, serverless functions, and static websites.
+
+You can do all of this without having to learn a DSL or script CLI commands. With Pulumi, developers and operators have a shared foundation for building modern applications.
+
+## How Pulumi . NET Works
+
+Pulumi is declarative even though it uses general purpose programming languages. It does this by declaring a set of resources and the Pulumi engine orchestrates the CRUD operations to build and deploy infrastructure.
+
+We use a . NET Core console application to declare infrastructure. Let's create an Azure App Service with a SQL database. The first step is make sure that your environment meets the prerequisites:
```

Comment:
Again, should be ".NET" no spaces

Reply:
removed, I think that the markdown formatter is doing this.